### ISSUE TITLE
Add a package logo?

### ISSUE THREAD

**Issue by IndrajeetPatil** [r-lib/vdiffr#123]
It'd be nice to have a hexsticker for the package, especially for people giving talks/presentations about this package or preparing social media posts.

**Comment by IndrajeetPatil** [comment]
How about something like this?

![vdiffr](https://user-images.githubusercontent.com/11330453/184112613-93fb4991-c3eb-4f5f-9b80-d930ef829eb8.png)

**Comment by lionel-** [comment]
Not bad. I think we'd get our hex designer on this though. We might also want a reference to testthat in the logo. @hadley do you think it's worth creating one?

**Comment by IndrajeetPatil** [comment]
> We might also want a reference to testthat in the logo.

Good idea. Something like in `{shinytest2}`'s [logo](https://rstudio.github.io/shinytest2/).
